apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/brev/model/BrevDTO.kt

```diff
@@ -0,0 +1,5 @@
+package no.nav.etterlatte.brev.model
+
+interface BrevDTO {
```

Comment:
Kan vurdere å bruke denne også for BrevData-klassane?

Reply:
Ja, tenker det burde være neste steg for å rydde litt opp etter jeg har fått inn denne. Må sette strek et sted 😛